client/app/stores/router_store.coffee

```diff
@@ -53,7 +53,7 @@ class RouterStore extends Store
 
     # Paginate Messages.list
     _pages = {}
-    _nextURL = {}
+    _requests = {}
```

Comment:
@aenario, @gregorylegarec, @m4dz : you can see that I rename variables to make obvious the relation between these and requestStore